kong/plugins/influxdb-log/schema.lua

```diff
@@ -0,0 +1,9 @@
+local utils = require "kong.tools.utils"
+local http_log_schema = require "kong.plugins.http-log.schema"
+
+local influxdb_log_schema = utils.deep_copy(http_log_schema)
+
+influxdb_log_schema.fields.method = { default = "POST", enum = { "POST" } }
```

Comment:
Check configuration items when use Restful Admin to config inflxudb-log. I want to return   response below when use the wrong configuration.

```
{"config.method":"\"GET\" is not allowed. Allowed values are: \"POST\""}

or

{"config.content_type":"\"application\/json\" is not allowed. Allowed values are: \"application\/x-www-form-urlencoded\""}
```